spec/browser/input_spec.rb

```diff
@@ -5,6 +5,7 @@
 module Ferrum
   describe "Browser::API::Input" do
     let!(:browser) { Browser.new(base_url: @server.base_url) }
+    let!(:select_all) { Ferrum.mac? ? %i[Alt shift Right] : %i[Ctrl A] }
```

Comment:
That shouldn't be with exclamation mark, because we don't need it to work as before hook. Just `let`